engage/app/styles/pages.less

```diff
@@ -45,7 +45,12 @@ a.list-group-item .list-group-item-heading.link-item-name {
 .media-item {
   padding: 15px;
 }
-/* calendar page */
-.calendar-item {
+
+.engage-media {
+  border-bottom: 1px solid #dddddd;
+  padding-bottom: 1.5rem;
```

Comment:
This code isn't necessary; just add `list-group-item engage-list-group-item` to the news item and you'll have the borders + padding without needing to write more css. The news items should be nearly identical HTML to the calendar items, shouldn't they?